src/rules/store/no-typed-global-store.ts

```diff
@@ -3,9 +3,9 @@ import { ESLintUtils } from '@typescript-eslint/experimental-utils'
 import path from 'path'
 import { docsUrl, typedStore } from '../../utils'
 
-export const messageId = 'noTypedStore'
+export const noTypedStore = 'noTypedStore'
```

Comment:
I took the opportunity to change this to keep it the same as we do in the other rules